src/page/modal-demo.js

```diff
@@ -33,6 +42,111 @@ export class ModalDemo extends React.Component {
+            <Col align='center'>
+              Update your information below
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <div className='col-md-6'>
+                <Input
+                  onChange={e => this.onTextChange('firstName', e)}
+                  label='First Name'
+                  value={this.state.firstName} />
+              </div>
+              <div className='col-md-6'>
+                <Input
+                  onChange={e => this.onTextChange('lastName', e)}
+                  label='Last Name'
+                  value={this.state.lastName} />
+              </div>
+            </Col>
+          </Row>
+        </Modal>
+
+        <Modal type='info' title='Add Link' buttonText='Save Changes' isOpen={this.state.customizedModal2} onClose={e => this.onCloseModal('customizedModal2')}>
+          <Row>
+            <Col>
+              Paste a link into the box below
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Input
+                onChange={e => this.onTextChange('link', e)}
+                placeholder='https://'
+                hasFeedbackIcon={false}
+                value={this.state.link}
+              />
+            </Col>
+          </Row>
+        </Modal>
+
+        <Modal type='success' title='Selectable Options' buttonText='Submit' isOpen={this.state.customizedModal3} onClose={e => this.onCloseModal('customizedModal3')}>
+          <Row>
+            <Col>
+              Select an option
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Select
+                placeholder='Options'
+                value={this.state.selectOne}
+                options={[ { value: 1, label: 'Example One' }, { value: 2, label: 'Example Two'}, { value: 3, label: 'Example Three'} ]}
+                onChange={value => this.setState({ selectOne: value })} />
+            </Col>
+          </Row>
+        </Modal>
+
+        <Modal type='danger' title='With Checkboxes' buttonText='Close' isOpen={this.state.customizedModal4} onClose={e => this.onCloseModal('customizedModal4')}>
+          <Row>
+            <Col>
+              True or False
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Input
+                type='radio'
+                name='firstSet'
+                label='True'
+                onChange={e => {}} />
+              <Input
+                type='radio'
+                name='firstSet'
+                label='False'
+                onChange={e => {}} />
+            </Col>
+          </Row>
+        </Modal>
+
+        <Modal type='warning' title='With Switches' buttonText='Save' isOpen={this.state.customizedModal5} onClose={e => this.onCloseModal('customizedModal5')}>
+          <Row>
+            <Col>
+              Notify me when...
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Col grow={false}>
```

Comment:
I don't think it works to have a `Col` inside a `Col` but it is ok to have a `Row`, then a `Col` with another set of Rows/Cols inside it